src/custom_nodes/common/customNodeLibraryInternalManager.cpp

```diff
@@ -0,0 +1,55 @@
+//*****************************************************************************
+// Copyright 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "customNodeLibraryInternalManager.hpp"
+
+#include <iostream>
+#include <string>
+
+namespace ovms {
+namespace custom_nodes_common {
+void CustomNodeLibraryInternalManager::createBuffersQueue(std::string name, size_t singleBufferSize, int streamsLength) {
+    auto it = outputBuffers.find(name);
+    if (it != outputBuffers.end()) {
+        delete it->second;
+        outputBuffers.erase(it);
```

Comment:
We need to explicitly call `delete` on it->second. It would not be neccessary if we had `map<std::string, BuffersQueue>`